frontend/public/locales/ja/pagerduty-receiver-form.json

```diff
@@ -20,5 +20,5 @@
   "Description": "説明",
   "Description of the incident.": "インシデントの説明。",
   "Severity": "重大度",
-  "Severity of the incident.": "インシデントの重大度。"
+  "Severity of the incident.": "インシデントの重大度。 "
```

Comment:
extra space added

Reply:
@rh-tchuang - Is it cool if we drop these extra spaces, or are they required for Japanese?

Reply:
@asasabris could you please confirm this for us? Thanks

Reply:
space taken

Reply:
Thanks! @sg00dwin; I'm going to resolve this one.